src/core_modules/capture-core/components/WidgetEnrollment/Actions/Delete/Delete.component.js

```diff
@@ -9,18 +10,21 @@ import {
     ButtonStrip,
     Button,
 } from '@dhis2/ui';
-import React, { useState } from 'react';
 import i18n from '@dhis2/d2-i18n';
 import type { Props } from './delete.types';
+import { ConditionalTooltip } from '../../../Tooltips/ConditionalTooltip/';
 
-export const Delete = ({ enrollment, onDelete }: Props) => {
+export const Delete = ({ canCascadeDeleteEnrollment, enrollment, onDelete }: Props) => {
     const [toggle, setToggle] = useState(false);
+    const disabled = useMemo(() => !canCascadeDeleteEnrollment, [canCascadeDeleteEnrollment]);
```

Comment:
Nit: I don't think we need to memoize this as it would just use up memory and the variable should be cached